scripts/utils.js

```diff
@@ -103,6 +104,21 @@ async function fetchWrapper(url, options) {
   return response.text()
 }
 
+async function findPackages() {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏it could be nice to have a naming a bit more specific:

- packages could easily be understood as the content of the `packages` directory
- it is kind of hard to guess what to expect as a result of this function

proposal: `generateMetadataForAllNodePackages()`